RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -541,6 +541,12 @@ extension OnLedgerEntitiesClient {
 	}
 }
 
+extension OnLedgerEntity.Resource {
+	var title: String {
+		metadata.name ?? metadata.symbol ?? L10n.TransactionReview.unknown
```

Comment:
This is not quite consistent with the [rule for resources display](https://radixdlt.atlassian.net/wiki/spaces/AT/pages/2844753933/Rules+for+Display+of+Assets+in+Account+Detail+View#Token-Display), according to which we should have something line `metadata.symbol ?? metadata.name`, if the resource has neither account or name, no title is shown.

is it meant to be used across the app?

Reply:
Yes that was the idea I suppose, I just moved it here from another location.